packages/core/src/tests/constants.ts

```diff
@@ -107,7 +107,9 @@ export enum ComponentDefaultTestId {
   MODAL_NEXT_OVERLAY = "modal-overlay",
   MODAL_NEXT_HEADER = "modal-header",
   MODAL_NEXT_CONTENT = "modal-content",
+  MODAL_NEXT_MEDIA = "modal-media",
   MODAL_NEXT_BASIC_LAYOUT = "modal-basic-layout",
+  MODAL_NEXT_SIDE_BY_SIDE_LAYOUT = "modal-side_by_side-layout",
```

Comment:
Why not kebab-case everything?

Reply:
it felt more right since modal is a prefix, layout is suffix, and the middle is its own context
you suggest changing it for consistency?

Reply:
IMO yes, I think it's less intuitive that way

Reply:
cool, will change